src/datascience-ui/variable-view/variableViewPanel.tsx

```diff
@@ -28,8 +28,6 @@ export class VariableViewPanel extends React.Component<IVariableViewPanelProps>
         super(props);
 
         this.panelRef = React.createRef<HTMLDivElement>();
-        // For the variable view we want to start toggled open
```

Comment:
Noticed that on view show an executionCount 0 message was getting sent first. Didn't really cause a bug (since it was immediately followed by the correct execution count fetch) but was unneeded and was due to this little hack. The toggle action triggered a fetch. Instead just added a way to specify on initial state if the view should be visible.